src/index.mjs

```diff
@@ -80,5 +175,8 @@ export const apolloUploadExpress = options => (request, response, next) => {
       request.body = body
       next()
     })
-    .catch(next)
+    .catch(error => {
+      if (error.status && error.expose) response.status(error.status)
```

Comment:
@AdamYee does this look like a good idea? Setting the response status automatically for errors in the Express middleware. I'm assuming it is an Express convention for middleware not to send the error response, that it is up to the consumer to set up a catch-all error handling middleware in their app?

Reply:
I think the simple `.catch(next)` is what you want. Inspecting the `error` object is usually the responsibility of the error handling middleware from my experience. They might choose to overwrite the status anyways, or not send an error at all.

I would take a look at multer's error handling API. https://github.com/expressjs/multer/blob/master/index.js#L40. Maybe there's value in providing a callback into your error handling in case the consumer wants to do something specific?